Code/GraphMol/DistGeomHelpers/BoundsMatrixBuilder.cpp

```diff
@@ -1734,6 +1730,18 @@ void collectBondsAndAngles(const ROMol &mol,
         tmp[1] = aid11;
         tmp[2] = aid21;
       }
+
+      if ((bondi->getBondType() == Bond::TRIPLE) ||
+          (bondj->getBondType() == Bond::TRIPLE)) {
```

Comment:
```suggestion
      if (bondi->getBondType() == Bond::TRIPLE ||
          bondj->getBondType() == Bond::TRIPLE) {
```
as you do not use parentheses in a similar `if` clause just below